cmd/crank/install.go

```diff
@@ -120,6 +129,30 @@ func (c *installConfigCmd) Run(k *kong.Context, logger logging.Logger) error {
 		logger.Debug("Failed to create configuration", "error", warnIfNotFound(err))
 		return errors.Wrap(warnIfNotFound(err), "cannot create configuration")
 	}
+	if err = waitUntil(waitDuration, func() error {
+		logger.Debug("Waiting for the configuration to be ready")
+		res, err := kube.Configurations().Get(context.Background(), pkgName, metav1.GetOptions{})
+		if err != nil {
+			logger.Debug("Failed to fetch created configuration", "error", err)
+			return errors.Wrap(err, "failed to fetch configuration")
+		}
+		var configStatus corev1.ConditionStatus
+		for _, condition := range res.Status.Conditions {
+			if condition.Type == v1.TypeHealthy {
+				configStatus = condition.Status
+			}
+		}
+		if configStatus == corev1.ConditionTrue {
+			logger.Debug("Configurtion is ready")
```

Comment:
```suggestion
			logger.Debug("Configuration is ready")
```

Reply:
Would also be good to move some of these messages to `const` values. Since they are used across both commands you could consider making them format strings so that you can pass "Configuration" or "Provider"